lib/common.js

```diff
@@ -25,13 +25,23 @@ module.exports.aliasResources = function (resourceObject) {
     services: ['svc']
   };
 
+  const esRemoval = [
```

Comment:
Rename `esRemoval` to `pluralsWithEs` or `esPlurals`

Also, make it an object.  this shorthand works in node.js >= 4.X
```
{
  componentstatuses,
  ingresses
}
```